### PR TITLE
Update as on 28-12-2020

### PR DESCRIPTION
**Pull request by mohsinmsm**
Update for Java Questions

### CODE REVIEW

File: java/java-quiz.md [hunk 1]
```diff
@@ -857,3 +857,51 @@ public class Solution {
 - [ ] instance
 - [X] class
 - [ ] method
+
+#### Q62. What does this code print?
+
+```
```
Comment (Ebazhanov) [inline java/java-quiz.md:863]:
```suggestion
```java
```

would be great to use java identification, which will highlight code with different color

File: java/java-quiz.md [hunk 2]
```diff
@@ -857,3 +857,51 @@ public class Solution {
...
+
+- [ ] 10 10
+- [X] 5 10
+- [ ] 10 5
+- [ ] 5 5
+
+#### Q63. What is the result of this code?
+
+```
```
Comment (Ebazhanov) [inline java/java-quiz.md:884]:
```suggestion
```java
```

the same here

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@mohsinmsm thanks for PR, left some comments

**Comment by mohsinmsm** [comment]
@Ebazhanov Kindly review the changes

**Review by Ebazhanov — APPROVED** [review]
Thank you @mohnishkarri, looks perfect now! Approved :punch:

**Comment by Ebazhanov** [comment]
@all-contributors please add @mohnishkarri for design, code

**Comment by mohsinmsm** [comment]
> Thank you @mohnishkarri, looks perfect now! Approved 👊

It's @mohsinmsm, I guess there's mistake in name

**Comment by Ebazhanov** [comment]
oh my bad sorry

**Comment by Ebazhanov** [comment]
@all-contributors please add @mohsinmsm for design, code

**Pull request merged**